registration.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓 - 로그인</title>
```

Comment:
❗️ 수정요청
로그인과 title이 서로 바뀐 것 같아요~

```suggestion
    <title>판다마켓 - 회원가입</title>
```